src/common/utility.js

```diff
@@ -211,61 +213,62 @@ const getDomain = () =>
 export {
     affiliate_signin_url,
     affiliate_signup_url,
+    application_id,
     applyDefaultValues,
     besquare_signup_url,
     binary_url,
+    blog_url,
     brand_name,
-    application_id,
-    client_token,
+    cfd_warning_height,
     checkElemInArray,
+    client_token,
     cloneObject,
-    blog_url,
-    cfd_warning_height,
-    deriv_cookie_domain,
-    dmt5_windows_url,
-    dmt5_linux_url,
-    dmt5_android_url,
-    dmt5_ios_url,
     community_url,
+    debounce,
     deriv_app_url,
     deriv_blog_url,
     deriv_bot_app_url,
+    deriv_cookie_domain,
```

Comment:
If all the conflicts are fixed, you can see, constants and utility functions are separated. You need import them from respected files.

Reply:
Resolved here https://github.com/binary-com/deriv-com/pull/1668